python/pyspark/pandas/tests/test_groupby.py

```diff
@@ -1257,7 +1258,34 @@ def _test_stat_func(self, func):
             (pdf.groupby("A"), psdf.groupby("A")),
             (pdf.groupby("A")[["C"]], psdf.groupby("A")[["C"]]),
         ]:
-            self.assert_eq(func(p_groupby_obj).sort_index(), func(ps_groupby_obj).sort_index())
+            self.assert_eq(
+                func(p_groupby_obj).sort_index(),
+                func(ps_groupby_obj).sort_index(),
+                check_exact=check_exact,
+            )
+
+    def test_basic_stat_funcs(self):
+        self._test_stat_func(lambda groupby_obj: groupby_obj.mean(), check_exact=False)
+        self._test_stat_func(lambda groupby_obj: groupby_obj.std(), check_exact=False)
```

Comment:
Hmm... seems like the `pdf.groupby("A")[["C"]].std()` raises Exception from pandas 1.4.0 as below:

```python
>>> pdf = pd.DataFrame(
...             {
...                 "A": [1, 2, 1, 2],
...                 "B": [3.1, 4.1, 4.1, 3.1],
...                 "C": ["a", "b", "b", "a"],
...                 "D": [True, False, False, True],
...             }
...         )
>>> pdf.groupby("A")[["C"]].std()
Traceback (most recent call last):
...
ValueError: could not convert string to float: 'a'
```

Can we fix & separate the test??

Reply:
Good catch! May I file a follow-up PR for the fix of `std`? Now tests are separated.

Reply:
Yeah, let's create ticket and follow it.